docs/source/developers/contributing/prerequisites.rst

```diff
@@ -34,6 +34,10 @@ This is the list of core dependencies to install for the most common tasks. In g
     .. code-block:: console
     
         $ apt install build-essential
+
+      Then install ``make``:
+  
+        $ apt install make
```

Comment:
```suggestion
    Then install ``make``:
  
    .. code-block:: console

        $ apt install make
```

Spacing is a little tricky for RST. Right now, what you currently have looks like this

<img width="2032" alt="Screenshot 2024-07-04 at 00 01 07" src="https://github.com/paraglider-project/paraglider/assets/12043327/94cb82e4-f358-4717-a548-56a49d11427b">

I assume you want it to look like this.

<img width="2032" alt="Screenshot 2024-07-04 at 00 01 15" src="https://github.com/paraglider-project/paraglider/assets/12043327/ff5a0ebc-b772-46c5-8d20-3102838fa7fa">